corehq/warehouse/transforms/sql/location_dim.sql

```diff
@@ -0,0 +1,97 @@
+-- First insert all records into location dim from staging tables without foreign keys
+INSERT INTO {{ location_dim }} (
```

Comment:
yeah coming in a later PR. right now it's blocking on `UPSERT`. i think it'll be fairly easy to convert